File: python/practices/fizzbuzz.py
start = 1
while start <= 50:
    if start%3 == 0 and start%5 == 0:
        print("fizzbuzz")
    elif start%5 == 0:
        print("buzz")
    elif start%3 == 0:
        print("fizz")
    else:
        print(start)
    start += 1

File: C/practices/fizzbuzz.c
#include <stdio.h> 
#include <string.h>
#include <stdbool.h>
#include <stdlib.h>
#include <time.h>


 int main(){
     
    for (int n = 1; n <= 51; n++){
        if (n%3 == 0 && n%5 == 0){
            printf("fizzbuzz\n");
        }else if (n%5 == 0){
            printf("buzz\n");
        }else if (n%3 == 0){
            printf("fizz\n");
        }else{
            printf("%d\n", n);
        } 
    }
    return 0;
}